input/provision_config.yml

```diff
@@ -36,24 +38,38 @@ admin_nic_subnet: "172.29.0.0"
 pxe_nic: "eno1"
 
 # The dhcp range for assigning the IPv4 address while provisioning
+# Examples: 172.29.0.50 - 172.29.0.100, 10.10.1.100 - 10.10.1.200
+# Default: 172.29.0.100 - 172.29.0.200
 # Mandatory
 pxe_nic_start_range: "172.29.0.100"
 pxe_nic_end_range: "172.29.0.200"
 
 ### IB NIC Details
 # If provided, omnia will handle and assign static IPs to compute node's IB network.
+# Only last 16 bits/2 octets of IPv4 are dynamic
+# Example: 172.29.0.0 , 10.10.0.0 , 192.15.0.0
+# If provided the db entry will be in parallel with either the pxe_nic ranges
+# Or whatever is provided in mapping file ( only if roce is disabled)
+# Example: If admin_ip: 172.29.0.101 and ib_nic_subnet: 178.30.0.0, then ib_ip: 178.30.0.101
 # Optional
 ib_nic_subnet: ""
 
 ### BMC NIC Details
-# If provided, omnia will handle and assign static IPs to the iDRACs.
+# If provided, Omnia will handle and assign static IPs to the iDRACs.
+# Only last 16 bits/2 octets of IPv4 are dynamic
+# Example: 172.29.0.0 , 10.10.0.0 , 192.15.0.0
```

Comment:
Can we have it different subnet here like this:
Examples: 172.28.0.0, 10.9.0.0, 192.14.0.0, note second octet of the subnet is different.

Reply:
Check once, I have updated it accordingly